docs/getting-started/apple/screenshots.md

```diff
@@ -1,8 +1,258 @@
+'/Users/mfurtak/.frameit/devices_frames', which should be open in your Finder
+You can just copy the whole content into it.
+Press Enter when you extracted the files into the given folder
+----------------------------------------------------
+```
+
+## Running
+
+At its simplest, you can just run:
+
+```sh
+fastlane frameit
+```
+
+A more complex example that configures a lot more about the output can be found in the [fastlane examples](https://github.com/fastlane/examples/tree/master/MindNode/screenshots) project.
+
+### Configure Using `Framefile.json`
+
+Your `Framefile.json` file should be in your `screenshots` folder, as seen in the [example](https://github.com/fastlane/examples/tree/master/MindNode/screenshots), and you can use it to define your configuration:
+
+```json
+{
+  "default": {
+    "keyword": {
+      "font": "./fonts/MyFont-Rg.otf"
+    },
+    "title": {
+      "font": "./fonts/MyFont-Th.otf",
+      "color": "#545454"
+    },
+    "background": "./background.jpg",
+    "padding": 50,
+    "show_complete_frame": false
+  },
+
+  "data": [
+    {
+      "filter": "Brainstorming",
+      "keyword": {
+        "color": "#d21559"
+      }
+    },
+    {
+      "filter": "Organizing",
+      "keyword": {
+        "color": "#feb909"
+      }
+    },
+    {
+      "filter": "Sharing",
+      "keyword": {
+        "color": "#aa4dbc"
+      }
+    },
+    {
+      "filter": "Styling",
+      "keyword": {
+        "color": "#31bb48"
+      }
+    }
+  ]
+}
+```
+
+The `show_complete_frame` value specifies whether _frameit_ should shrink the device and frame so that they show in full in the framed screenshot. If it is false, then they can hang over the bottom of the screenshot.
+
+The `filter` value is a part of the screenshot named for which the given option should be used. If a screenshot is named `iPhone5_Brainstorming.png` the first entry in the `data` array will be used.
+
+You can find a more complex [configuration](https://github.com/fastlane/examples/blob/master/MindNode/screenshots/Framefile.json), which also supports Chinese, Japanese, and Korean languages.
+
+### Localizing With `.strings` Files
+
+To define the title and optionally the keyword, put two `.strings` files into the language folder (e.g. [en-US in the example project](https://github.com/fastlane/examples/tree/master/MindNode/screenshots/en-US))
+
+The `keyword.strings` and `title.strings` are standard `.strings` file you already use for your iOS apps, making it easy to use your existing translation service to get localized titles.
+
+**Note:** These `.strings` files **MUST** be utf-16 encoded (UTF-16 LE with BOM).  They also must begin with an empty line. If you are having trouble see [issue #1740](https://github.com/fastlane/fastlane/issues/1740)
+
+**Note:** You **MUST** provide a background if you want titles. _frameit_ will not add the tiles if a background is not specified.
+
+# Upload Screenshots for the App Store
+
+TODO: write about _deliver_
```

Comment:
This should probably describe mostly the screenshots part of deliver, and not the binary and metadata part, right? We should reference the other parts from here too 👍